src/runtime/node/vm/script.ts

```diff
@@ -0,0 +1,16 @@
+import type vm from "node:vm";
+
+export class Script implements vm.Script {
+  runInContext(
+    contextifiedObject: vm.Context,
+    options?: vm.RunningScriptOptions | undefined,
+  ) {}
```

Comment:
I think we should throw here. I can't imagine a scenario where it would be acceptable for the implementation to be a noop.